### PR TITLE
DOC: Add bootstrap hint for Leap 15.0

### PR DESCRIPTION
**Pull request by asdil12**
Ticket: https://progress.opensuse.org/issues/45365

### CODE REVIEW

File: docs/Installing.asciidoc [hunk 1]
```diff
@@ -34,12 +34,24 @@ zypper in openQA-bootstrap
 /usr/share/openqa/script/openqa-bootstrap
 -------------------------------------------------------------------------------
 
+If you happen to be using an old Leap 15.0 system which doesn't yet have the
```
Comment (Martchus) [inline docs/Installing.asciidoc]: I would just write 'If you still use an old Leap 15.0 system which doe**s no**t **already** have the ...'

File: docs/Installing.asciidoc [hunk 2]
```diff
@@ -34,12 +34,24 @@ zypper in openQA-bootstrap
...
+
 === openQA in a container
 
 You can also setup a systemd-nspawn container with openQA with the following
-commands. This will only work on Tumbleweed and you need to have no application
-listening on port 80 yet because the container will share the host systems
-network stack.
+commands. *This will only work on Tumbleweed* (or at least not before Leap 15.1)
+and you need to have no application listening on port 80 yet because the container
+will share the host systems network stack.
```
Comment (Martchus) [inline docs/Installing.asciidoc]: s/systems/system's/

Comment (okurz) [reply docs/Installing.asciidoc]: what? why?

File: docs/Installing.asciidoc [hunk 3]
```diff
@@ -34,12 +34,24 @@ zypper in openQA-bootstrap
...
+If you happen to be using an old Leap 15.0 system which does not already have the
+openQA-bootstrap RPM in the repo you can simply download the openqa-bootstrap
+script - it will do the rest for you:
+
+[source,sh]
+-------------------------------------------------------------------------------
+# get root
+wget https://raw.githubusercontent.com/os-autoinst/openQA/master/script/openqa-bootstrap
+chmod +x openqa-bootstrap
+./openqa-bootstrap
```
Comment (okurz) [inline docs/Installing.asciidoc]: Could you please make this a one-line call?

File: docs/Installing.asciidoc [hunk 4]
```diff
@@ -34,12 +34,24 @@ zypper in openQA-bootstrap
...
+
 === openQA in a container
 
 You can also setup a systemd-nspawn container with openQA with the following
-commands. This will only work on Tumbleweed and you need to have no application
-listening on port 80 yet because the container will share the host systems
-network stack.
+commands. *This will only work on Tumbleweed* (or at least not before Leap 15.1)
+and you need to have no application listening on port 80 yet because the container
+will share the host system's network stack.
```
Comment (okurz) [inline docs/Installing.asciidoc]: Now it's wrong, I am pretty sure

File: docs/Installing.asciidoc (hunk 2 above)
Comment (Martchus) [reply docs/Installing.asciidoc]: So this is plural?

**Pull request merged**